plugins/plugin-chart-echarts/src/Tree/controlPanel.tsx

```diff
@@ -0,0 +1,327 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { t } from '@superset-ui/core';
+import { ControlPanelConfig, sections, sharedControls } from '@superset-ui/chart-controls';
+import { DEFAULT_FORM_DATA } from './types';
+
+const requiredEntity = {
+  ...sharedControls.entity,
+  clearable: false,
+};
+const optionalEntity = {
+  ...sharedControls.entity,
+  clearable: true,
+  validators: [],
+};
+
+const controlPanel: ControlPanelConfig = {
+  controlPanelSections: [
+    sections.legacyRegularTime,
+    {
+      label: t('Query'),
+      expanded: true,
+      controlSetRows: [
+        [
+          {
+            name: 'id',
+            config: {
+              ...requiredEntity,
+              label: t('Id'),
+              description: t('Name of the id column'),
+            },
+          },
+        ],
+        [
+          {
+            name: 'parent',
+            config: {
+              ...requiredEntity,
+              label: t('Parent'),
+              description: t('Name of the column containing the id of the parent node'),
+            },
+          },
+        ],
+        [
+          {
+            name: 'name',
+            config: {
+              ...optionalEntity,
+              label: t('Name'),
+              description: t('Optional name of the data column.'),
+            },
+          },
+        ],
+        [
+          {
+            name: 'root_node_id',
```

Comment:
Didn't set renderTrigger true because chart gives error like node is not defined or bugs out with weird connections like below.
![Screenshot_2021-04-15  DEV  Tree](https://user-images.githubusercontent.com/12967587/114913955-67691400-9e3f-11eb-93ca-ca108eda585b.png)